tests/drivers/dac/dac_api/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  tags: dac drivers userspace
+tests:
+  drivers.dac:
+    platform_whitelist: nucleo_l073rz
```

Comment:
Can we make this `dt_compat_enabled_with_alias` look at samples/basic/threads/sample.yaml for example.

Reply:
That won't be sufficient. The source file contains board-specific code.

Reply:
So I assume we can keep this as it is?